docs/sovereign-cloud/README.md

```diff
@@ -0,0 +1,67 @@
+# Sovereign Cloud Guide for Project Symphony
+
+This document highlights the steps necessary to test this project with a sovereign cloud.
+
+# OPTIONAL: Register new cloud
+
+0. Before running ```az cloud show``` to export the current cloud's details, run it once to view the current cloud details in the Terminal. If necessary, run ```az cloud set``` to set the desired cloud as the active cloud.
+    * ```az cloud show -o table```
+
+1. Run the following command to export the current cloud's details into a .json file
+    * ```az cloud show > cloud.json```
+
+1. Delete the following values from the generated .json file: isActive, name and profile, since these 3 values will be automatically assigned during the ```az cloud register``` process.
+
+1. Run the following command, using the revised .json file:
+    * ```az cloud register -n USSec --cloud-config @"cloud.json"```
+
+1. Log out of current cloud
+    * ```az logout```
+
+1. Set newly registered cloud
+    * ```az cloud set -n USSec```
```

Comment:
Definitely do NOT mention a specific cloud.. use AzureStack instead.

Reply:
Good catch, fixed.